lib/app.rb

```diff
@@ -44,7 +37,11 @@ class App < Sinatra::Base
   # Search for Manifests
   get '/api/:version/manifest/search' do
     content_type :json
-    Manifest.all.to_json
+    if !params[:q] && !params[:aq]
```

Comment:
search applications historically use the `q` param for the query string. I try to keep with that convention. Here are some reference documents.

https://www.elastic.co/guide/en/elasticsearch/reference/current/search-uri-request.html
https://wiki.apache.org/solr/CommonQueryParameters

My understanding is that short URL parameter names are used because Back In the Day there were limits to URL length and each extra byte in the URL required that much more bandwidth.